### ISSUE TITLE
[Audio] Set Tag Master Volume

### ISSUE THREAD

**Issue by OverboyDev** [Scirra/Construct-feature-requests#17]
### Reviewed guidelines

- [X] I have read and understand the suggestion guidelines

### Checked for duplicate suggestions

- [X] I checked for existing similar suggestions

### Summary

**"Set tag master volume" action (taking a percent param from 0 to 100%)** 
to change the sound of all currently played and upcoming sounds of a specific tag

### Possible workarounds or alternatives

**ISSUE 1** The new multiple audio tags feature is nice, but it actually still doesn't help that much to set up a SFX and a Music sliders options because right now, if we set volume by tag, it just overrides the volumes all the sound tagged with "sfx" and/or "music". But it isn't really the intended behavior, as each of those sounds could have a different individual volume.

For example if I have a huge explosion with the tag "explosion sfx" (db=0) and a footstep with the tag "footstep sfx" (db=-20). If I set the "sfx" tag volume to db = -5, then it would override the value of both the footstep and the explosion, so the footstep can be even louder than the explosion !

**ISSUE 2** Also if i play a "sfx" sound AFTER i set up the "sfx" tag volume, then this last sound isn't affected by the tag volume, as it just overrides the currently playing sounds.

### Proposed solution

TLDR : **A Set Tag Master Volume (Percentage) Action**

So the thing C3 is lacking to achieve that would be **a new "set tag master volume" action**, so we can lower the volume of all "musics" or "sfxs" at once, but without overriding their individual values, and it would work even for the sounds that aren't playing yet but that will be played after changing the tag master volume.

**This way each sound would have a volume based on their individual initial value + the existing Global Master Volume + this new Tag-based Master Volume action.**

**It is important that this feature supports any object tag (not just the first tag), because the first tag is usually the most specific (as it's used for effects), while secondary tags are perfect to categorize sounds in bigger groups. ("sfx" or "music" for example).
It also means the sound can be linked to several master volumes.**

For example i can have a "explosion outside sfx" with initial volume value set to W, and then set "sfx" master volume to X, set "outside" master volume to Y and set the existing Global Master volume to Z. The actual volume of my sound tagged "explosion outside sfx" would depend on W, X, Y and Z.

**It could be cool if the Set Master Volume action takes a percentage parameter** and C3 automatically handle the db conversion under the hood. As it doesn't make sense to remove X db to every sound starting from different Y db values **as db isn't linear**

### Why is this idea important?

Most games have Sound and Music Volume sliders, while it's incredibly hard to implement in C3

(The SFX/Music master are the most common example as it's needed in literally any games, but there would be plenty of other usecases for this)

**Comment by alastajj** [comment]
I'm still keen to see this

**Comment by alastajj** [comment]
@OverboyDev  @F3der1co @Everade @XHXIAIEIN @Wackytoaster @brushfe do you think we still need this feature?
